commit d24b21e394580ad92e3a633f3588eb45143f8040 (refs/changes/62/5262/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-12-18T12:14:52-08:00 (10 months ago)
    
    TRITON-1046 move SAPI's audit logger to a shareable triton-audit-logger package (add new repo to repos list)

diff --git a/repos.json b/repos.json
index 8ca69fb..06c4386 100644
--- a/repos.json
+++ b/repos.json
@@ -313,6 +313,9 @@
         {
             "name": "node-triton"
         },
+        {
+            "name": "node-triton-audit-logger"
+        },
         {
             "name": "node-triton-merci-buckets"
         },
@@ -886,6 +889,7 @@
             "name": "triton-grafana",
             "labels": {
                 "tritonservice": "grafana",
+                "mg": "grafana",
                 "vm": true
             }
         },
@@ -893,6 +897,7 @@
             "name": "triton-kbmapi",
             "labels": {
                 "tritonservice": "kbmapi",
+                "mg": "kbmapi",
                 "vm": true
             }
         },
@@ -915,6 +920,7 @@
             "name": "triton-prometheus",
             "labels": {
                 "tritonservice": "prometheus",
+                "mg": "prometheus",
                 "vm": true
             }
         },
@@ -970,7 +976,6 @@
         "conch-locker",
         "conch-reporter-smartos",
         "conch-shell",
-        "conch-stats",
         "conch-ui",
         "consul",
         "containerpilot",
